### PR TITLE
Revert "Use tap.remote for pull_pr for Linuxbrew"

### PR DESCRIPTION
**Pull request by dunn**
Follow-up to #35; see https://bot.brew.sh/job/Homebrew%20Bottles/65369/console:
```
brew pull --clean https://github.com/Homebrew/homebrew-emacs.git/pull/515
Error: Not a GitHub pull request or commit: https://github.com/Homebrew/homebrew-emacs.git/pull/515
```

### CODE REVIEW

File: cmd/brew-test-bot.rb [hunk 1]
```diff
@@ -984,7 +984,7 @@ def test_ci_upload(tap)
     # These variables are for Jenkins and Circle CI respectively.
     pr = ENV["UPSTREAM_PULL_REQUEST"] || ENV["CIRCLE_PR_NUMBER"]
     if pr
-      pull_pr = "#{tap.remote}/pull/#{pr}"
+      pull_pr = "#{tap.remote.sub(/\.git$/, "")}/pull/#{pr}"
```
Comment (dunn) [inline cmd/brew-test-bot.rb]: Well, what was the benefit to Linuxbrew of the new version? @sjackman?

Comment (MikeMcQuaid) [inline cmd/brew-test-bot.rb]: I don't think `test-bot` needs to handle that edge case any more given recent discussions about Linuxbrew/homebrew-science.

Comment (sjackman) [inline cmd/brew-test-bot.rb]: True, at least for me. Why would Homebrew support atypical git remotes but not `test-bot`? The pair of #35 and #42 implement this support. I'm fine without out it, just curious why.

Comment (MikeMcQuaid) [inline cmd/brew-test-bot.rb]: > True, at least for me.

Sorry, does that mean you no longer need this functionality? Thanks!

Comment (sjackman) [inline cmd/brew-test-bot.rb]: Correct. Though I am curious why Homebrew supports atypical git remotes and `homebrew-test-bot` does not, I can live without this functionality.

Comment (sjackman) [inline cmd/brew-test-bot.rb]: Thanks for the explanation, Mike.

Comment (MikeMcQuaid) [inline cmd/brew-test-bot.rb]: NP!

Comment (dunn) [inline cmd/brew-test-bot.rb]: OK, I'll update this PR to just revert #35.

**Comment by dunn** [comment]
Can I go ahead and merge this?

**Comment by MikeMcQuaid** [comment]
Thanks @dunn!
